src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setNumber(new Random().nextInt(100));
+        ball.setColor(new ColorSupplier().getRandomColor());
```

Comment:
make global Random and ColorSupplier fields and use them here, so you won't need to create their instances each time when you call this method